Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source 'https://rubygems.org'
 
-ruby '~> 3.3.5'
+ruby '~> 3.4.1'
```

Comment:
WDYT about
`ruby file: '.ruby-version'` ?

https://bundler.io/guides/gemfile_ruby.html
https://andycroll.com/ruby/read-ruby-version-in-your-gemfile/